### PR TITLE
Feature: API metadata and Execute via LSP

### PR DESCRIPTION
**Pull request by LinceMathew**
## What type of MR is this? (check all applicable)

- [x] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Doc Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] ⏩ Revert

## Description

This merge request includes two major changes.
1. Added API response time, status code, size of API in bytes for all API responses.
2. We can execute an API request via LSP now, that will return the API response as a RPC Json along with the above mentioned meta data.

## Important files to start review from
-  [lsp_execute.go](https://github.com/HexmosTech/Lama2/blob/feature/meta-data/l2lsp/methods/lsp_execute.go)
- [output_manager.go](https://github.com/HexmosTech/Lama2/blob/feature/meta-data/outputManager/output_manager.go)

## Mobile & Desktop Screenshots/Recordings

image

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## If documentation update is there, then run `make mkdocs` once PR is in acceptable state.

- [ ] 📓 make mkdocs
- [x] 📜 README.md
- [ ] 🙅 no documentation needed

## [optional] Are there any post-deployment tasks we need to perform?
- Update documentation
- Update readme
## [optional] What gif best describes this MR or how it makes you feel?

alt text

### CODE REVIEW

File: cmdexec/cmdexec.go [hunk 1]
```diff
@@ -21,13 +22,18 @@ import (
 // to stdout.
 // Once execution finishes, previous CWD is restored,
 // and the command output is returned as a string
-func ExecCommand(cmdSlice []string, stdinBody string, apiDir string) (httpie.ExResponse, error) {
+func ExecCommand(cmdSlice []string, stdinBody string, apiDir string) (httpie.ExResponse, int64,  error) {
 	oldDir, _ := os.Getwd()
 	utils.ChangeWorkingDir(apiDir)
+	start := time.Now()
```
Comment (rogueloop) [inline cmdexec/cmdexec.go:28]: We are checking time http request only ?

File: cmdexec/js.wasm.go [hunk 2]
```diff
@@ -30,7 +30,7 @@ func RunVMCode(jsCode string, vm interface{}) {
 
 // GenerateChainCode takes in an HTTP response body and comes up with some JS code to define the "magic variable" result.
 func GenerateChainCode(httpRespBody string) string {
-	fmt.Println("WW: Value of httpRespBody is",httpRespBody)
+	// fmt.Println("WW: Value of httpRespBody is",httpRespBody)
```
Comment (rogueloop) [inline cmdexec/js.wasm.go:33]: remove comments

File: controller/controller.cli.go [hunk 3]
```diff
@@ -23,9 +24,9 @@ import (
 	"github.com/rs/zerolog/log"
 )
 
-func HandleParsedFile(parsedAPI *gabs.Container, o *lama2cmd.Opts, dir string) (httpie.ExResponse, *lama2cmd.Opts) {
-	vm := cmdexec.GetJSVm()
-	return HandleParsedFileHelper(parsedAPI, o, dir, vm)
+func HandleParsedFile(parsedAPI *gabs.Container, o *lama2cmd.Opts, dir string) (httpie.ExResponse, *lama2cmd.Opts, []outputmanager.ResponseTime, []outputmanager.StatusCode, []outputmanager.ContentSize, error) {
+    vm := cmdexec.GetJSVm()
+    return HandleParsedFileHelper(parsedAPI, o, dir, vm)
```
Comment (rogueloop) [inline controller/controller.cli.go:29]: run `go fmt ./...`

File: cmdexec/cmdexec.go (hunk 1 above)
Comment (LinceMathew) [reply cmdexec/cmdexec.go:28]: yes  the time difference between executing the API request

File: cmdexec/js.wasm.go (hunk 2 above)
Comment (LinceMathew) [reply cmdexec/js.wasm.go]: removed

File: controller/controller.cli.go (hunk 3 above)
Comment (LinceMathew) [reply controller/controller.cli.go]: Addressed

File: .github/workflows/internal_release.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    - uses: wangyoucao577/go-release-action@v1.32
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        goos: ${{ matrix.goos }}
+        goarch: ${{ matrix.goarch }}
+        goversion: "https://dl.google.com/go/go1.19.3.linux-amd64.tar.gz"
+        binary_name: "l2"
+        extra_files: LICENSE README.md
+        ldflags: -X main.version=${{  github.ref_name }}
+        build_flags: '-tags=cli'
```
Comment (rogueloop) [inline .github/workflows/internal_release.yml:33]: add wasm build here its in `release.yml`

File: codegen/codegen.wasm.go [hunk 5]
```diff
@@ -60,7 +60,7 @@ func generateConvertedSippet(targetLangLib string, parsedAPI *gabs.Container) st
 			snippetArgs.SnippetCore = snippetcore
 			fmt.Println("Snippet args:", snippetArgs)
 			httpsnippetCode := PrepareHTTPSnippetGenerator(snippetArgs)
-			//fmt.Println("httpsnippetCode:", httpsnippetCode)
+			// fmt.Println("httpsnippetCode:", httpsnippetCode)
```
Comment (rogueloop) [inline codegen/codegen.wasm.go:63]: remove these comments

File: .github/workflows/internal_release.yml (hunk 4 above)
Comment (LinceMathew) [reply .github/workflows/internal_release.yml]: addressed

File: codegen/codegen.wasm.go (hunk 5 above)
Comment (LinceMathew) [reply codegen/codegen.wasm.go]: done

**Pull request merged**